src/sdk/model/vpg/json/vpg-failover-create-request.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * Vpg Failover Create Request properties
+ */
+export interface VpgFailoverCreateRequest {
```

Comment:
Names of all interfaces in the `json` folder should end in 'Json', and all should be wrapped by a class at the SDK level. Use Nico's pattern for request classes that we talked about as a group.

Reply:
Ho yeah. Good catch !